spec/models/concerns/journaled/actor_spec.rb

```diff
@@ -31,15 +31,15 @@ def trigger_before_actions
   it "Stores a thunk returning nil if current_user returns nil" do
     subject.trigger_before_actions
 
-    allow(subject).to receive(:current_user).and_return(nil)
+    allow(subject).to receive(:current_user).and_return(nil) # rubocop:disable RSpec/SubjectStub
 
     expect(RequestStore.store[:journaled_actor_proc].call).to eq nil
   end
 
   it "Stores a thunk returning current_user if it is set when called" do
     subject.trigger_before_actions
 
-    allow(subject).to receive(:current_user).and_return(user)
+    allow(subject).to receive(:current_user).and_return(user) # rubocop:disable RSpec/SubjectStub
```

Comment:
This is mocking the unit under test, but I think these cases are ok so I ignored em!

Reply:
yeah, lgtm. this is basicaly post hoc dependency injection, not really stubbing the SUT. At least that's how I sleep at night. ;)